setup.py

```diff
@@ -32,7 +32,6 @@
         "django-ipware>=1.1.6,<1.2",
         "future==0.16.0",
         "requests",
-        "six",
```

Comment:
I think we may then need to change the import paths in the code to import from Django's vendored version, right?

Reply:
I grepped and I think I got the only one. It's possible I missed one.

Reply:
I mistakenly didn't get that in the same commit though

Reply:
Ah looks like I did miss two imports :man_facepalming: Fixing!